addons/calendar/models/calendar_attendee.py

```diff
@@ -114,7 +114,10 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
                 event_id = attendee.event_id.id
                 ics_file = ics_files.get(event_id)
 
-                attachment_ids = mail_template.attachment_ids.ids
+                if mail_template.attachment_ids:
+                    attachment_ids = mail_template.attachment_ids.copy({'res_id': 0, 'res_model': 'mail.compose.message'}).ids
```

Comment:
copy in a loop == :bomb: 

Can you do `copy_data` before the loop and create as many as self in one batch?

Reply:
not sure how can I do it, can you say more about it?